golang/pkg/clusterpb/cluster.pb.go

```diff
@@ -1412,7 +1427,7 @@ var file_cluster_proto_rawDesc = []byte{
 	0x6c, 0x65, 0x12, 0x0a, 0x0a, 0x06, 0x4c, 0x45, 0x41, 0x44, 0x45, 0x52, 0x10, 0x00, 0x12, 0x0c,
 	0x0a, 0x08, 0x46, 0x4f, 0x4c, 0x4c, 0x4f, 0x57, 0x45, 0x52, 0x10, 0x01, 0x42, 0x36, 0x5a, 0x34,
 	0x67, 0x69, 0x74, 0x68, 0x75, 0x62, 0x2e, 0x63, 0x6f, 0x6d, 0x2f, 0x43, 0x65, 0x72, 0x65, 0x73,
-	0x44, 0x42, 0x2f, 0x63, 0x65, 0x72, 0x65, 0x73, 0x64, 0x62, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x2f,
+	0x44, 0x42, 0x2f, 0x68, 0x6f, 0x72, 0x61, 0x65, 0x64, 0x62, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x2f,
```

Comment:
I'm not sure if this a breaking change and I have no idea why the result change.

Reply:
@ShiKaiWi @jiacai2050   Please help to check it.

Reply:
`0x43, 0x65, 0x72, 0x65, 0x73, 0x44, 0x42, 0x2f, 0x63, 0x65, 0x72, 0x65, 0x73, 0x64, 0x62, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x2f, 0x67, 0x6f, 0x6c, 0x61, 0x6e, 0x67, 0x2f,` means `CeresDB/ceresdbproto/golang/`(hex to utf-8).

Reply:
Never mind, those auto-generated bytes have no real effect. 

Explain below are generated by Google bard.

> The proto_rawDesc in Golang
 is a byte array that represents the serialized form of a protocol
 buffer message. It is generated by the protoc compiler
 from a .proto file. The proto_rawDesc can be used to create a new protocol buffer message, or it can be deserialized into an existing message.
To create a new protocol buffer message from the proto_rawDesc, you can use the NewMessage() function. This function takes the proto_rawDesc as its argument and returns a new protocol buffer message.
To deserialize the proto_rawDesc into an existing message, you can use the Parse() function. This function takes the proto_rawDesc as its argument and parses it into the existing message.
The proto_rawDesc is a useful way to store and transmit protocol buffer messages. It is a compact and efficient way to represent the data, and it is easy to serialize
 and deserialize.